README.rst

```diff
@@ -111,11 +58,12 @@ User can override envrionment variables to force it reset and rebuilt before pro
     export CONTAINERS_TO_DESTROY='all_containers:!galera_all:!neutron_agent:!ceph_all:!rsyslog_all'
```

Comment:
I think we probably still need this for the situation where we don't care about the Elastic Search and want to remove it.  The CONTAINERS_TO_DESTORY variable is used in OSA_OPS and UPGRADE_ELASTICSEARCH is used in post leap so it's used in two different locations.  

By default it will save and upgrade, but if you don't want to save the data, you'd set upgrade to No and destroy the elasticsearch container so that it would be rebuilt from scratch again.  If you remove this override, the container data would remain and it would attempt to configure it fresh but with a preexisting container.